third_party/tls_cert_test/README.MD

```diff
@@ -0,0 +1,45 @@
+# TLS self-signed certificates for integration tests.
+
+**Valid until**: Jun 16 13:48:30 2999 GMT
+
+**Generate on**: Apr 15 13:48:30 2019 GMT
+
+**Generated with the script from**: https://github.com/grpc/grpc-java/tree/master/examples/example-tls
+
+```bash
+mkdir -p /tmp/sslcert
+pushd /tmp/sslcert
+# Changes these CN's to match your hosts in your environment if needed.
+SERVER_CN=localhost
+CLIENT_CN=localhost # Used when doing mutual TLS
+
+echo Generate CA key:
+openssl genrsa -passout pass:1111 -des3 -out ca.key 4096
+echo Generate CA certificate:
+# Generates ca.crt which is the trustCertCollectionFile
+openssl req -passin pass:1111 -new -x509 -days 365 -key ca.key -out ca.crt -subj "/CN=${SERVER_CN}"
```

Comment:
365 days?

Reply:
It doesn't matter actually, the flag -days from openssl x509 command - this is what sets up the valid until time